bundler/lib/bundler/plugin/installer.rb

```diff
@@ -45,20 +46,35 @@ def check_sources_consistency!(options)
         if options.key?(:git) && options.key?(:local_git)
           raise InvalidOption, "Remote and local plugin git sources can't be both specified"
         end
+        options[:git] = options.delete(:local_git) if options.key?(:local_git)
```

Comment:
It the `--local_git` option now a loop? I guess we should keep it working as before for now.

Reply:
it simply renames the option to :git here, before choosing the install method. the old install_git and install_local_git methods were identical (except choosing a different option key). so actual functionality has not changed

Reply:
Oh, my bad, I read the code incorrectly, thanks for clarifying!